frame/composable-support/src/validation.rs

```diff
@@ -113,11 +188,22 @@ pub(crate) mod private {
 	}
 }
 
-impl<T: codec::Encode + codec::Decode + Validate<U>, U> codec::WrapperTypeEncode
+impl<T: codec::Encode + codec::Decode, U: Validate<T, U>> codec::WrapperTypeEncode
 	for Validated<T, U>
 {
 }
 
+// impl<T: codec::Encode + codec::Decode, U: Validate<T, U>> codec::WrapperTypeDecode
```

Comment:
A marker trait for types that can be created solely from other decodable types.

The decoding of such type is assumed to be the same as the wrapped type.

If we comment that out, we may fail to have goo TS scale API

Reply:
This wasn't removed from the existing code. This is a leftover from me trying to fix an issue that I resolved in a different way. This has been removed here 005472a15e3a48707e9a81b975df0cf6f9ae0a8d